src/auro-menu.js

```diff
@@ -490,6 +484,9 @@ class AuroMenu extends LitElement {
       });
 
       this.notifyReady();
+    } else {
+      // make sure to update all menuoption noCheckmark attributes when the menu is dynamically changed
+      this.handleNoCheckmarkAttr();
```

Comment:
See above comment... `noCheckmark` also needs to work for single level menus.

Reply:
This one, I honestly forgot why I put it inside the `else` condition.
Should I put it outside the condition, so it always be called like how it was supposed to be then?

Reply:
I just found the reason on why I put in the else condition, because `this.handleNoCheckmarkAttr();` is already called in `initItems()` inside  the `if (this.rootMenu)` condition, otherwise it would be called twice.
What do you think ?
```

/**
   * Initializes all menu options in the DOM. This must be re-run every time the options are changed.
   * @private
   */
  initItems() {
    this.items = Array.from(this.querySelectorAll('auro-menuoption'));
    this.handleNoCheckmarkAttr();
  }
```